ops/ansible/roles/bfd-pipeline/templates/bfd-pipeline-service.sh.j2

```diff
@@ -42,6 +42,9 @@ export RDA_JOB_WRITE_THREADS='{{ data_pipeline_rda_job_write_threads }}'
 {% if data_pipeline_rda_process_dlq is defined %}
 export RDA_JOB_PROCESS_DLQ='{{ data_pipeline_rda_process_dlq }}'
 {% endif %}
+{% if data_pipeline_rda_version is defined %}
+export RDA_JOB_RDA_VERSION='{{ data_pipeline_rda_version }}'
+{% endif %}
```

Comment:
Overall, I think this PR's description could better serve everyone with a little extra detail. Perhaps the minor deviation from the stated requirements under PACA-199 (_The RDA Version will be a required environment variable, with no default_) might be a good seam to look a little further. To be clear, I think your solution is better 😉 

Checking my understanding here of the proposed change... are the following statements accurate?
* the environment variable `RDA_JOB_RDA_VERSION` functions as an external, operational override
* as an override value, it will not have a _sensible_ default
* when this environment variable is absent, the application yields to the version encoded in the application's `RdaService` `RDA_PROTO_VERSION` field

If the above is accurate, I have just two lingering questions if you wouldn't mind entertaining them:
* glancing at the history of the `RDA_PROTO_VERSION` field, it doesn't look like this has been updated as frequently as the overall changes supporting updates/upgrades i.e. obviously, this change set updates the value from 0.10.0 to 0.12.0, but the RDA-based pipeline jobs already support 0.12.0. Overall, does this change set introducing the RDA API Version check make `RDA_PROTO_VERSION` a more meaningful setting?
* I'm largely ambivalent, but I do favor solutions that allow for configuration like this one. I'm curious what  a _most likely_ scenario might be where we would want to override the value of the RdaService's `RDA_PROTO_VERSION` field with the content of `RDA_JOB_RDA_VERSION`. Is this potentially to support incremental patch versions and the like?

Reply:
You are right on the `RDA_JOB_RDA_VERSION` being an override feature, the primary mechanism is the internal hardcoded value.  This was a bit of an evolution on the concept after the ticket was written, with the argument being that the code is really tied to a specific RDA version (with certain transformations needing to exist, etc.), so hardcoding it seemed more appropriate, but we wanted to leave a potential hook if the need arrises for some reason.

As for the `RDA_PROTO_VERSION` hardcoded in the app, previously that was used for local testing, acting as the version used by the mock RDA API we use.  The actual value this was set to did not actually affect anything I don't think, we just tried to keep it in sync with the actual API for testing reasons.  Now it's being used as the default value for `RDA_JOB_RDA_VERSION` (which again, is the preferred way to get this value), so it's value is more important now.  It would not be necessary to update it on every API update, as you noticed we skipped over 0.11.0 because that update didn't involve any actual changes to the API messages.

The way we use it is also more like version ratcheting, where the API can go ahead and release 0.14.0 whenever, as long as it's not a breaking change (which minor changes should not be breaking), and we should still be happy consuming it with our service set to allow `^0.12.0`.  If catastrophe happened on the upstream API though and they slipped back to like `0.11.0`, it would stop that from being consumed.

To be honest I don't have a good use case off hand for the external override of the env variable, more one of those, just in case you need it, situations.  It also makes it a little more consistent configuration wise, while we are _technically_ hardcoding the value, the configuration is loaded just like every other configuration, which also just makes it a bit more flexible of a design.

Reply:
OK. Thanks, Kyle. I'll meditate on how I'd want to capture some of this as _knowledge_. I may come back to edit the content of this PR's body, but I'lll go ahead and approve for now!